src/test/java/uk/gov/hmcts/cmc/claimstore/services/CountyCourtJudgmentServiceTest.java

```diff
@@ -107,7 +109,31 @@ public void saveShouldFinishCCJByAdmissionSuccessfullyForHappyPath() {
 
         verify(eventProducer, once()).createCountyCourtJudgmentEvent(any(Claim.class), any());
         verify(claimService, once()).saveCountyCourtJudgment(eq(AUTHORISATION), any(), any());
-        verify(appInsights, once()).trackEvent(eq(AppInsightsEvent.CCJ_REQUESTED),
+        verify(appInsights, once()).trackEvent(eq(AppInsightsEvent.CCJ_REQUESTED_BY_ADMISSION),
+            eq(REFERENCE_NUMBER), eq(claim.getReferenceNumber()));
+    }
+
+    @Test
+    public void saveShouldFinishCCJAfterBreachSuccessfullyForHappyPath() {
```

Comment:
FinishCCJAfterBreach..HappyPath sounds like DC comic superhero finishing villain and happy ever after. Please rename e.g. shouldSaveCCJAfterBreachOfSettlementAgreement?